test/controllers/update/relevant.period.owned.land.filter.spec.ts

```diff
@@ -139,12 +145,20 @@ describe("owned land filter page tests", () => {
       expect(resp.header.location).toEqual(config.RELEVANT_PERIOD_INTERRUPT_URL);
     });
     test(`renders the ${config.UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL} page when no is selected`, async () => {
+      mockGetApplicationData.mockReturnValue({ ...APPLICATION_DATA_MOCK });
+      mockIsActiveFeature.mockReturnValueOnce(true);
       const resp = await request(app)
         .post(config.RELEVANT_PERIOD_OWNED_LAND_FILTER_URL)
         .send({ owned_land_relevant_period: "0" });
+      const mockAppData = {};
 
+      mockAppData[ChangeBoRelevantPeriodKey] = undefined;
```

Comment:
What is this testing? do we need `mockAppData` here?

Reply:
modified the code, thanks Connor